src/celeritas/phys/ParticleData.hh

```diff
@@ -19,36 +19,24 @@ namespace celeritas
 //---------------------------------------------------------------------------//
 // PARAMS
 //---------------------------------------------------------------------------//
-/*!
- * Fundamental (static) properties of a particle type.
- *
- * These should only be fundamental physical properties. Setting particles is
- * done through the ParticleParams. Physical state of a particle
- * (kinetic energy, ...) is part of a ParticleState.
- *
- * Particle definitions are accessed via the ParticleParams: using PDGs
- * to look up particle IDs, etc.
- */
-struct ParticleRecord
-{
-    units::MevMass mass;  //!< Rest mass [MeV / c^2]
-    units::ElementaryCharge charge;  //!< Charge in units of [e]
-    real_type decay_constant;  //!< Decay constant [1/s]
-    bool is_antiparticle;  //!< Antiparticle (negative PDG number)
 
-    //! Value of decay_constant for a stable particle
-    static CELER_CONSTEXPR_FUNCTION real_type stable_decay_constant()
-    {
-        return 0;
-    }
+enum class MatterType : char
+{
+    particle,
+    antiparticle
 };
 
+//! Value of decay_constant for a stable particle
+constexpr real_type stable_decay_constant = 0;
```

Comment:
Maybe this should go in `celeritas/Constants.hh`?

Reply:
Also this is missing an `inline`—otherwise it won't work with anything that takes `real_type const&` (which often shows up in templated containers for example).